.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+version: 2
+updates:
+  - package-ecosystem: nuget
```

Comment:
Perhaps a different package ecosystem? Also, because we don't want to maintain this as frequently, let's change the interval to monthly instead of weekly.